### ISSUE TITLE
Disable Screen Time problems

### ISSUE THREAD

**Issue by Panak252** [leminlimez/Cowabunga#285]
So, I was experimenting with the new options, and I gave "Disable Screen Time" a try. I did have wifi and the sort off, and it was working. That's not the issue. The issue is, it doesn't revert itself. I've tried rebooting, shutdowns, changing the switch from off to on, and back to off. Still hasn't reverted. I enabled Screen Time again, but whenever an app reaches its limit, the phone freezes for a minute or two, and then resprings back into my lockscreen. If I go to the Settings app, and then tap on Screen Time, the app does the same thing. I can exit Settings at will, but if I don't, it resprings.

![image](https://user-images.githubusercontent.com/88540067/233066257-4abd93e0-1c00-4426-aa31-54ac2350f49a.png)

In conclusion, the "Disable Screen Time" should have an undo option, or a different switch to re-enable it again.

iPhone 7+, iOS 15.6.1, Cowabunga 10.1.1 Release (Newest available as of currently)

**Comment by hellojsna** [comment]
Same on iOS 16.1.2(iPhone 12 Pro).
It stops working after launching Cowabunga app.
